src/urllib3/_private_compat/rlock.py

```diff
@@ -0,0 +1,12 @@
+"""
```

Comment:
Adding a new module requires a change to `setup.py:packages=[...]`, why not `urllib3._compat`?

Reply:
Got it—you mean a single `_compat.py` file at the top level?

Reply:
Yeah!